invenio_search/__init__.py

```diff
@@ -279,6 +270,51 @@ def index():
 
 .. _invenio-access:  https://invenio-access.readthedocs.io/
 
+Elasticsearch version support
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Invenio-Search currently supports Elasticsearch versions 2.x and 5.x. The only
```

Comment:
Perhaps a quick note saying ES6 support is coming as soon as DSL library is out (v6 support is already in master branch so likely it's soon)